frontend/src/components/archive/ArchiveListItem.tsx

```
@@ -0,0 +1,165 @@
+            >
+              View
+            </Link>
+            {onDelete && (
+              <button
+                onClick={handleDeleteClick}
+                onKeyDown={handleKeyDown}
+                className="px-4 py-2 border border-red-300 text-red-600 rounded text-sm font-medium hover:bg-red-50 focus:outline-none focus:ring-2 focus:ring-red-500 focus:ring-offset-2 transition-colors"
+                aria-label={`Delete archive ${name}`}
+                disabled={isDeleting}
+                aria-busy={isDeleting}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)